blog/2022/10-20-resoto-cdk/index.md

```diff
@@ -0,0 +1,43 @@
+---
+authors: [nikita]
+tags: [aws, cdk, installation]
+---
+
+# Resoto AWS CDK construct
+
+We recently released a [AWS CDK construct](https://eu-central-1.console.aws.amazon.com/cloudformation/home?region=eu-central-1#/stacks/create/review?templateURL=https://resoto-cdk.s3.eu-central-1.amazonaws.com/Resoto_EKS.template&stackName=ResotoEKS), which simplifies the deployment of Resoto to AWS.
```

Comment:
Hmm, this link is not accessible when you are not logged in. Is there another link we can use here?

Reply:
I'm afraid not.

Reply:
good catch. I will update it